x/migration/module/genesis.go

```diff
@@ -0,0 +1,26 @@
+package migration
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/pokt-network/poktroll/x/migration/keeper"
+	"github.com/pokt-network/poktroll/x/migration/types"
+)
+
+// InitGenesis initializes the module's state from a provided genesis state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+    // this line is used by starport scaffolding # genesis/module/init
```

Comment:
whitespace fix

Reply:
I'm not sure what the ask is here, could you please elaborate?

Reply:
Tabs vs spaces 😅 

Just saw the `arrow` vs `...` as I was scrolling.


![Screenshot 2025-02-03 at 11 12 45 AM](https://github.com/user-attachments/assets/ea417278-dac3-4ef8-8b34-12cead2617b1)

Reply:
Feel free to ignore